Support/support-de.json

```diff
@@ -185,33 +185,33 @@
-                    "text": "Your AdGuard Home is probably only accessible from your local network (so when you're at home). The best way to be able to access & use your AdGuard Home from outside your network as well, is to run a VPN. I can recommend looking into [PiVPN](https://pivpn.io) or [Tailscale](https://tailscale.com). That way you can safely access your AdGuard Home from anywhere!"
+                    "title": "Warum werden die Daten nicht geladen, wenn ich nicht zu Hause bin?",
+                    "text": "Dein AdGuard Home ist wahrscheinlich nur von Deinem lokalen Netzwerk aus zugänglich (also wenn Du zu Hause sind). Die beste Möglichkeit, auf Dein AdGuard Home auch von außerhalb Deines Netzwerks zuzugreifen und ihn zu nutzen, ist die Verwendung eines VPN. Ich kann Dir [PiVPN] (https://pivpn.io) oder [Tailscale] (https://tailscale.com) empfehlen. Auf diese Weise kannst Du von überall aus sicher auf Dein AdGuard Home zugreifen!"
                 },
                 {
                     "id": 2,
-                    "title": "Why is my homescreen widget not showing data?",
-                    "text": "Make sure you are currently on a network where your device can reach your AdGuard Home instance, such as your home wifi. You can also try a device restart which will restart the widget completely."
+                    "title": "Warum zeigt mein Homescreen-Widget keine Daten an?",
+                    "text": "Vergewissere Dich, dass Du Dich in einem Netzwerk befindest, in dem Dein Gerät Deine AdGuard Home-Instanz erreichen kann, z. B. in Deinem Heim-WLAN. Du kannst auch einen Neustart des Geräts durchführen, um das Widget komplett neu zu starten."
                 },
                 {
                     "id": 3,
-                    "title": "Why is the Apple Watch app stuck on a spinner?",
-                    "text": "First, make sure you are logged into iCloud on your iPhone. Then restart your phone & watch, open the iPhone app, then open the Watch app. If that doesn't work, a fresh reinstall of the app should work. Sorry for the inconvenience!"
+                    "title": "Warum bleibt die Apple Watch App an einem Ladekreis hängen?",
+                    "text": "Stelle zunächst sicher, dass Du bei iCloud auf Deinem iPhone angemeldet bist. Starte dann Dein Telefon und Deine Uhr neu, öffne die iPhone-App und dann die Watch-App. Wenn das nicht funktioniert, sollte eine Neuinstallation der App helfen. Ich entschuldige mich für die Unannehmlichkeiten!"
                 },
                 {
                     "id": 4,
-                    "title": "I'm using Home Assistant, how do I setup the app?",
-                    "text": "In Home Assistant > Configuration > Add-ons, Backups & Supervisor > AdGuard Home > Configuration, there is a field for 'Network Host'. Assign a port-number there for the web interface and enter it in the app."
+                    "title": "Ich verwende den Home Assistant. Wie richte ich die App ein?,
```

Comment:
Small mistake here I think, values should always start and end with `"` and end with a `,` if another key-value pair comes behind it

Reply:
I'll fix this and push the fix myself 😄 so I can make a new beta build now

Reply:
Oh I am sorry. I accidentally deleted a " ... Sorry !:D
Was ist just in this line?

Reply:
Yes was just here 😄 but I fixed it with a separate commit